scout/server/blueprints/panels/views.py

```diff
@@ -384,12 +384,14 @@ def gene_edit(panel_id, hgnc_id):
     form.disease_associated_transcripts.choices = tx_choices(hgnc_id, panel_obj)
     if form.validate_on_submit():
         action = "edit" if panel_gene else "add"
-        info_data = form.data.copy()
-        if "csrf_token" in info_data:
-            del info_data["csrf_token"]
-        if info_data["custom_inheritance_models"] != "":
-            info_data["custom_inheritance_models"] = info_data["custom_inheritance_models"].split(
-                ","
+        info_data: dict = {
+            k: v for k, v in form.data.copy().items() if v
```

Comment:
This is the fix

Reply:
But, we have booleans there? What f the user sets a value of e.g. mosaicism to `False`?

Reply:
Yes, it's a checkbox (boolean). So on or off. One could refactor and allow it to be reset to None, but I don't want to do it in this PR. I think it's best to assume on is True (so mosaicism or reduced penetrance) and off is those things unknown. For now at least. These checkbox are not really used much I think, otherwise somebody would have complained already

Reply:
Sorry, I'm writing too tersely.

The thing with that is you can in the main version unset a `Yes` once you have it set (perhaps mistakenly, perhaps since literature changed). This PR will not allow it:
![Screenshot 2024-10-11 at 08 19 00](https://github.com/user-attachments/assets/8b82ac20-2456-49fa-95d2-456603931d14)
![Screenshot 2024-10-11 at 08 22 17](https://github.com/user-attachments/assets/8c1db5f7-0be5-4ce8-9b65-609ea73c6bce)
![Screenshot 2024-10-11 at 08 22 21](https://github.com/user-attachments/assets/0cce0e4c-08d4-4639-8b93-fb641483e187)
<img width="222" alt="Screenshot 2024-10-11 at 08 21 53" src="https://github.com/user-attachments/assets/2d4a42c1-d160-477c-989f-8dd10d116978">
![Screenshot 2024-10-11 at 08 19 28](https://github.com/user-attachments/assets/a552efc6-0682-4e94-badf-cca67b5eb158)
since a False or a None value in the update is treated the same way.